packages/release-please/src/release-please.ts

```diff
@@ -225,6 +225,64 @@ async function buildManifest(
   );
 }
 
+async function runBranchConfiguration(
+  github: GitHub,
+  repoLanguage: string | null,
+  repoUrl: string,
+  branchConfiguration: BranchConfiguration,
+  logger: GCFLogger
+) {
+  let manifest = await buildManifest(
+    github,
+    repoLanguage,
+    branchConfiguration,
+    logger
+  );
+
+  // release-please can handle creating a release on GitHub, we opt not to do
+  // this for our repos that have autorelease enabled.
+  if (branchConfiguration.handleGHRelease) {
+    logger.info(`handling GitHub release for (${repoUrl})`);
+    try {
+      const numReleases = await Runner.createReleases(manifest);
+      logger.info(`Created ${numReleases} releases`);
+      if (numReleases > 0) {
+        // we created a release, reload config which may include the latest
+        // version
+        manifest = await buildManifest(
+          github,
+          repoLanguage,
+          branchConfiguration,
+          logger
+        );
+      }
+    } catch (e) {
+      if (e instanceof Errors.DuplicateReleaseError) {
+        // In the future, this could raise an issue against the
+        // installed repository
+        logger.warn('Release tag already exists, skipping...', e);
```

Comment:
Perhaps I am missing something, but should you return here?

Reply:
No, this has 2 steps, attempting to tag releases (if any), then maybe opening a new release pull request. We are explicitly ignoring duplicate tag errors from tagging and will continue on to try opening new pull requests